test/com/amazon/ion/impl/bin/IonEncoder_1_1Test.java

```diff
@@ -205,6 +219,186 @@ public void testWriteFloatValueForDouble(double value, String expectedBytes) {
+            "2023-01T,                            01110001 10110101 00000000",
+            "2023-10T,                            01110001 00110101 00000101",
+            "2023-12T,                            01110001 00110101 00000110",
+            "2023-10-01T,                         01110010 00110101 00001101",
+            "2023-10-15T,                         01110010 00110101 01111101",
+            "2023-10-31T,                         01110010 00110101 11111101",
+            "2023-10-15T01:00-00:00,              01110011 00110101 01111101 00000001 00000000",
+            "2023-10-15T01:59-00:00,              01110011 00110101 01111101 01100001 00000111",
+            "2023-10-15T11:22-00:00,              01110011 00110101 01111101 11001011 00000010",
+            "2023-10-15T23:00-00:00,              01110011 00110101 01111101 00010111 00000000",
+            "2023-10-15T23:59-00:00,              01110011 00110101 01111101 01110111 00000111",
+            "2023-10-15T11:22:00-00:00,           01110100 00110101 01111101 11001011 00000010 00000000",
+            "2023-10-15T11:22:33-00:00,           01110100 00110101 01111101 11001011 00010010 00000010",
+            "2023-10-15T11:22:59-00:00,           01110100 00110101 01111101 11001011 10110010 00000011",
+            "2023-10-15T11:22:33.000-00:00,       01110101 00110101 01111101 11001011 00010010 00000010 00000000",
+            "2023-10-15T11:22:33.444-00:00,       01110101 00110101 01111101 11001011 00010010 11110010 00000110",
+            "2023-10-15T11:22:33.999-00:00,       01110101 00110101 01111101 11001011 00010010 10011110 00001111",
+            "2023-10-15T11:22:33.000000-00:00,    01110110 00110101 01111101 11001011 00010010 00000010 00000000 00000000",
+            "2023-10-15T11:22:33.444555-00:00,    01110110 00110101 01111101 11001011 00010010 00101110 00100010 00011011",
+            "2023-10-15T11:22:33.999999-00:00,    01110110 00110101 01111101 11001011 00010010 11111110 00001000 00111101",
+            "2023-10-15T11:22:33.000000000-00:00, 01110111 00110101 01111101 11001011 00010010 00000010 00000000 00000000 00000000",
+            "2023-10-15T11:22:33.444555666-00:00, 01110111 00110101 01111101 11001011 00010010 01001010 10000110 11111101 01101001",
+            "2023-10-15T11:22:33.999999999-00:00, 01110111 00110101 01111101 11001011 00010010 11111110 00100111 01101011 11101110",
+    })
+    public void testWriteTimestampValueWithUnknownOffsetShortForm(@ConvertWith(StringToTimestamp.class) Timestamp value, String expectedBytes) {
+        assertWritingValueWithBinary(expectedBytes, value, IonEncoder_1_1::writeTimestampValue);
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+            //                                    OpCode   MYYYYYYY DDDDDMMM mmmHHHHH ooooommm ssssssoo ffffffff ffffffff ffffffff ..ffffff
+            "2023-10-15T01:00-14:00,              01111000 00110101 01111101 00000001 01000000 00000010",
```

Comment:
Should the offset bits all be zero for this one?

Reply:
Why yes, it should. 😅 
I've gone and manually double checked several more test cases with offsets both in the short form and the long form timestamps. I forgot about adjusting the offset for the short form, but the long form looks okay.